build/myget-release.yml

```diff
@@ -11,6 +11,8 @@ variables:
     value: '2.2.105'
   - name: 'Project'
     value: 'Arcus.Security'
+  - name: 'AzureServicesAuthConnectionString'
```

Comment:
Do you need this one perse? I'll add it to variable group for sake of being sure

Reply:
Yeah, docs says:

> or set an environment variable named **AzureServicesAuthConnectionString** to **RunAs=App;AppId=AppId;TenantId=TenantId;AppKey=Secret**. You need to replace **AppId**, **TenantId**, and **Secret** with actual values.

Reply:
👌 Added to variable group so feel free to remove it here